subsys/bluetooth/audio/Kconfig.mcs

```diff
@@ -14,6 +14,7 @@ config BT_MCS
 	bool "Media Control Service Support"
 	select BT_CCID
 	select EXPERIMENTAL
+	depends on BT_L2CAP_TX_BUF_COUNT >= 10
```

Comment:
Instead add `default 10  if BT_MCS`  in `subsys/bluetooth/host/Kconfig.l2cap` under `BT_L2CAP_TX_BUF_COUNT`

Reply:
I disagree. If MCS require the BUF_COUNT to be 10 to work properly (which I guess is the case?), then `default` should not be used. `BT_L2CAP_TX_BUF_COUNT` can still be set to e.g. 2, and then you have a MCS that doesn't work as intended.

Reply:
with `depends` `BT_MCS` is not available for users. Try using `ninja menuconfig` and you will not see MCS listed under audio services.

Reply:
I see. I had not considered how `depends on` affect `ninja menuconfig` (I always just edit those in the text files instead...).

Reply:
@asbjornsabo Should we add a BUILD_ASSERT for mcs.c to ensure that there's enough L2CAP buffers?

Reply:
Added